translation/en/RedirectedResources/assets/abdata/adv/scenario/c34/20/105/translation.txt

```diff
@@ -1,45 +1,45 @@
-//「Ｗｏｗ……Ｉ……Ｉ’ｍ　ｓｐｅｅｃｈｌｅｓｓ……」=
-//「スピーチ、レス？」=
-//「ア……エット……キレイすぎて……なんて表現したらイイカ、ワカラナイカラ……♡」=
-//「ああ、だからスピーチレスなんだ。英語にもそういう表現があるんだね」=
-//「ウン……トッテモ、キレイ……ココ、誰もいないシ……アナタとワタシ、世界に二人キリ、なったミタイ……」=
-//「[H名]、だいぶ日本語うまくなったね」=
-//「……ウン、ワタシ、ニッポン語、トッテモ上達したヨ。その理由、分かる？」=
-//「え？　えーと……[H名]がいっぱい勉強したから、だよね？　図書室で本読んだり、国語の勉強に苦戦したり、頑張ってたよね」=
-//「フフ♡　ソレダケ？　モットモット、上達する理由、あるデショ？」=
-//「……えーと……ちょっと恥ずかしいけど……俺、かな？」=
-//「Ｙｅｓ♡　アノネ……帰国する飛行機デ、Ｍａｍａがワタシに言ったノ。『ニッポン語が上手くなりたいなら、恋をするのが一番』ッテ」=
-//「恋は、好きなヒトとココロを合わせたい気持ちデ、言葉は、ココロを合わせる恋のＣｕｐｉｄダッテ……Ｍａｍａの言う通りだったヨ♡」=
-//「[H名]のママも、そうだったのかな」=
-//「ソウダヨ♡　ワタシが恋をする『オトシゴロ』ナノニ、今でもＰａｐａと毎日Ｌｏｖｅ　ｌｏｖｅダモン♡　コッチがハズカシ～ってなるノ♡」=
-//「デモ……Ｍａｍａが『今でも、パパに恋してる』って言うと、アナタのコト、思うノ……ウウン、想う……」=
+﻿「Ｗｏｗ……Ｉ……Ｉ’ｍ　ｓｐｅｅｃｈｌｅｓｓ……」=Wow... I... I'm speechless...
+「スピーチ、レス？」=Speech, less?
+「ア……エット……キレイすぎて……なんて表現したらイイカ、ワカラナイカラ……♡」=Ah... Erm... It's just too pretty... I don't know what sort of expression would fit here...♡
+「ああ、だからスピーチレスなんだ。英語にもそういう表現があるんだね」="Aah, that's why you were speechless". English has those sorts of expressions too, huh?""
+「ウン……トッテモ、キレイ……ココ、誰もいないシ……アナタとワタシ、世界に二人キリ、なったミタイ……」=Yeah... It's, really beautiful... There's no one else here either... It feels like we're the only ones, in the world...
+「[H名]、だいぶ日本語うまくなったね」=Your Japanese has gotten really good, [H名].
+「……ウン、ワタシ、ニッポン語、トッテモ上達したヨ。その理由、分かる？」=...Yeah, I've gotten really good at Japanese. Do you understand why?
+「え？　えーと……[H名]がいっぱい勉強したから、だよね？　図書室で本読んだり、国語の勉強に苦戦したり、頑張ってたよね」=Eh? Erhm... It's because you studied a lot, no? Studying books all day in the library, struggling against it the best you could, you worked really hard, didn't you?
+「フフ♡　ソレダケ？　モットモット、上達する理由、あるデショ？」=Fufu♡ Is that all? There should be a way more important reason, to why I got so good at it, no?
+「……えーと……ちょっと恥ずかしいけど……俺、かな？」=...Erhm... This sounds a bit embarrassing coming from my mouth but... Is it, me?
+「Ｙｅｓ♡　アノネ……帰国する飛行機デ、Ｍａｍａがワタシに言ったノ。『ニッポン語が上手くなりたいなら、恋をするのが一番』ッテ」="Yes♡ You know... When we were on the plane back to Japan, my Mama said to me: If you want to get good at Japanese, fall in love with someone."""
+「恋は、好きなヒトとココロを合わせたい気持ちデ、言葉は、ココロを合わせる恋のＣｕｐｉｄダッテ……Ｍａｍａの言う通りだったヨ♡」="Love is the feeling of wanting to be one of heart with the person you like, and words are the cupids of love that match those hearts"... It was just as Mama said ♡""
+「[H名]のママも、そうだったのかな」=I wonder if your Mama was the same way, [H名].
+「ソウダヨ♡　ワタシが恋をする『オトシゴロ』ナノニ、今でもＰａｐａと毎日Ｌｏｖｅ　ｌｏｖｅダモン♡　コッチがハズカシ～ってなるノ♡」=She was♡ Even after I got to the age of loving someone myself, she still spends all day being lovey-dovey with Papa♡ I feel secondhand embarrassment everytime♡
+「デモ……Ｍａｍａが『今でも、パパに恋してる』って言うと、アナタのコト、思うノ……ウウン、想う……」="But... everytime Mama says Even now, I love Papa", I think of you... No, I 'think' of you... ""
 //「英語は思考の"思う"モ、想像の"想う"モ、ｔｈｉｎｋダケド、アナタにワタシの恋、伝えるためにニッポン語、勉強したヨ。ワタシの"オモウ"、ドッチ？　分かる？」=
```

Comment:
Did something get missed by the tool when merging?

Reply:
Seems fixed